ansible/library/droplet_vlan_cfg.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+import yaml
+import traceback
+
+DOCUMENTATION = '''
+module: droplet_vlan_cfg.py
+Ansible_version_added:  2.0.0.2
+short_description: Gather all vlan info related to droplet interfaces 
+Description:
+       When deploy testbed topology with droplet connected to TOR SONiC, 
+       gather droplet vlan interfaces info for generating SONiC minigraph file
+ arguments:
+    vm_topo_config: Topology file; required: True
+    port_alias: Port aliases of TOR SONiC; required: True
+    vlan_cfg:  vlan config name to use; required: False
+
+Ansible_facts:
+    'vlan_cfgs': all Vlans Configuration 
+'''
+
+EXAMPLES = '''
+    - name: find all vlan configurations for T0 topology
+      droplet_vlan_cfg: 
+        vm_topo_config: "{{ vm_topo_config }}"
+        port_alias: "{{ port_alias }}"
+        vlan_cfg: "{{ vlan_cfg|default(None) }}"
+'''
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            vm_topo_config=dict(required=True),
+            port_alias=dict(required=True),
```

Comment:
port_aliases?

Reply:
This mirrored the name provided by the ansible module port_alias:

https://github.com/Azure/sonic-mgmt/blob/master/ansible/library/port_alias.py#L131
https://github.com/Azure/sonic-mgmt/blob/master/ansible/config_sonic_basedon_testbed.yml#L73